### PR TITLE
Spartan fluff

### PR DESCRIPTION
**Pull request by Draconic-saint**
Fluff item for LP Spartan: minimalist gasmask that's designed for use in tandem with their custom sec-hud.

Item icon. (That I totally didn't forget and made in dm)
dreammaker_2017-02-24_02-41-02

Looks like this when equiped.
photoshop_2017-02-24_02-40-08

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -552,6 +552,13 @@
 	desc = "It's a worn-out bandana in camo paint"
 	icon_state = "bandcamo"
 
+
+/obj/item/clothing/mask/gas/sechailer/fluff/spartan //LP Spartan: Kaskreyarawkta
+	name = "minimal gasmask"
+	desc = "Designed to cover as little of face as possible while still being a functional gasmask.
```
Comment (FlattestGuitar) [inline code/modules/customitems/item_defines.dm]: "

**Comment by FlattestGuitar** [comment]
Also, probably wanna restrict this to Vox, since there are no sprites for Humans and everyone else.

**Comment by Fox-McCloud** [comment]
@FreeStylaLT @Necaladun

**Comment by FreeStylaLT** [comment]
👌

**Comment by KasparoVy** [comment]
Gotta say, looks pretty sick!

**Pull request merged**